apps/console/src/public/deployment.config.json

```diff
@@ -350,7 +350,6 @@
             },
             "applications": {
                 "disabledFeatures": [
-                    "applications.loginFlow.ai",
                     "applications.loginFlow.legacyEditor"
```

Comment:
Hope we've sent a `carbon-identity-framework` PR?

Reply:
Sent with https://github.com/wso2/carbon-identity-framework/pull/6338